src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  die() {
+    const index = Animal.alive.indexOf(this);
+
+    Animal.alive.splice(index, 1);
```

Comment:
Using indexOf and splice within the die method creates a nested loop which can be avoided for better performance. Consider using the filter method to create a new array without the dead animal.